api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -208,6 +208,8 @@
     String DEFAULT_CLIENT_PROXY_HOST_PROPERTY_NAME = "okta.client.proxy.host";
     String DEFAULT_CLIENT_PROXY_USERNAME_PROPERTY_NAME = "okta.client.proxy.username";
     String DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME = "okta.client.proxy.password";
+    String DEFAULT_CLIENT_RETRY_MAX_ELAPSED_PROPERTY_NAME = "okta.client.requestTimeout";
```

Comment:
Nit: the config var is now `requestTimeout`, but we are still using "max elapsed" in property names and method names